devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy

```diff
@@ -145,7 +145,7 @@ def parse(String description) {
 			result = zwaveEvent(cmd)
 		}
 	}
-	if (result?.name == 'hail' && hubFirmwareLessThan("000.011.00602")) {
+	if (result?.name? == 'hail' && hubFirmwareLessThan("000.011.00602")) {
```

Comment:
I don't believe this change will have any effect. DId you see an error from this?

Reply:
`timestamp=2020-03-17 05:44:56.215, level=WARN, log_id=3571100b-15ba-4e81-a17c-72af5e342125, trace_id=, user=, cat=DeviceTypeExecutionService, key=deviceType_error, deviceId=828121d8-e0f5-47e3-8fe4-9629fc70332f, hubId=59974d3d-0754-4450-a686-28116956ba9f, username=user_uuid:bf5bda9e-9483-597d-79fd-5b1a5e194654, isLogLevelDebugEnabled=false, checksum=1bfe4865c2c2c3ee50a5122025aaba2b769733032b8ad675beb7d2799f96a989, deviceTypeId=ea21ba57-8406-40d0-ac01-c4e46b95dc1b, deviceTypeName=Z-Wave Dimmer Switch Generic, namespace=smartthings, state=LATEST_APPROVED, type=parse, runTime=25,
groovy.lang.MissingPropertyException: Exception evaluating property 'name' for java.util.ArrayList, Reason: groovy.lang.MissingPropertyException: No such property: name for class: physicalgraph.device.HubAction
	at org.kohsuke.groovy.sandbox.impl.Checker$4.call(Checker.java:115)
	at physicalgraph.sandbox.SmartAppSandbox.onGetProperty(SmartAppSandbox.groovy:362)
	at org.kohsuke.groovy.sandbox.impl.Checker$4.call(Checker.java:113)
	at org.kohsuke.groovy.sandbox.impl.Checker.checkedGetProperty(Checker.java:110)
	at script_dth_1bfe4865c2c2c3ee50a5122025aaba2b769733032b8ad675beb7d2799f96a989.parse(script_dth_1bfe4865c2c2c3ee50a5122025aaba2b769733032b8ad675beb7d2799f96a989:148)`

so in retrospect most likely not the right fix, but I did see it

Reply:
in double-retrospect, this looks like the right fix

this error is thrown in response to the `stopMultilevelChange` command, since it generates two events.

Reply:
I also see this is specific to very old hub firmware so can probably be completely removed at some point but looks fine for the time being.